index.html

```diff
@@ -985,55 +985,56 @@ <h2>Preliminaries</h2>
-                    a <a>Vocabulary Term</a> in the <a>Signature</a> of the <a>Class</a> is mandatory.
-                    <a>Vocabulary Terms</a> that are not mandatory, but are in the <a>Signature</a>
-                    of a <a>Class</a> are optional.
+                    if for every <a>Term</a> for which the <a>Assignment Function</a> of the <a>Class</a>
+                    returns <code>true</code>, the <a>Object</a> then includes a name-value pair with the
+                    <a>Vocabulary Term</a> as name.
                 </li>
                 <li>
-                    an <a>Object</a> is an instance of a <a>Class</a>
-                    if for every <a>Vocabulary Term</a>
-                    in the <a>Signature</a> of the <a>Class</a> used as name in some name-value pair
-                    of the <a>Object</a>, the value of that pair is an instance of the
-                    <a>Class</a> returned by the <a>Type Function</a> of the <a>Class</a> for the
-                    given <a>Vocabulary Term</a>.
+                    if for every <a>Vocabulary Term</a> in the <a>Signature</a> of the <a>Class</a>
+                    used as name in some name-value pair of the <a>Object</a>, the value of that pair
+                    is an instance of the <a>Class</a> returned by the <a>Type Function</a> of the
+                    <a>Class</a> for the given <a>Vocabulary Term</a>.
                 </li>
             </ul>
         </p>
 
         <p>
             According to the definition above, an <a>Object</a> would be an instance of every <a>Simple
             Type</a>, regardless of its structure. Instead, another definition for the <a>Instance
-            Relation</a> is introduced for <a>Simple Types</a>:
-            <ul>
-                <li>
-                    An <a>Object</a> is an instance of a <a>Simple Type</a> if it is a <a>Term</a>
-                    with a given lexical form (e.g. <code>true</code>, <code>false</code> for the
-                    <code>boolean</code> type, <code>1</code>, <code>2</code>, <code>3</code>, ...
-                    for the <code>unsignedInt</code> type, etc).
-                </li>
-            </ul>
+            Relation</a> is introduced for <a>Simple Types</a>: an <a>Object</a> is an instance
+            of a <a>Simple Type</a> if it is a <a>Term</a> with a given lexical form (e.g. <code>true</code>,
+            <code>false</code> for the <code>boolean</code> type, <code>1</code>, <code>2</code>,
+            <code>3</code>, ... for the <code>unsignedInt</code> type, etc).
         </p>
 
         <p>
-            A <a>Class</a> is a <dfn>Subclass</dfn> of some other <a>Class</a> if every
-            instance of the former is also an instance of the latter.
+            Moreover, special <a>Classes</a> can be derived from the generic <a>Map</a> and <a>Array</a>
+            structures: an <a>Object</a> is a <a>Map</a> (respectively, <a>Array</a>) of some <a>Class</a>,
```

Comment:
respectively -> alternatively

Reply:
removed all parentheses and added "The same applies to Arrays".